Makefile

```diff
@@ -91,8 +96,8 @@ update-teleport-repo:
 	cd teleport; git checkout $(TELEPORT_TARGET) || git checkout -b $(TELEPORT_TARGET)
```

Comment:
if we expect to run it locally, then it would be better to use 1 temporary directory instead of 2: `teleport` and `dist`.

We can then delete this line `echo teleport >> .gitignore` from this script (and add this line to webassets .gitignore file instead)